src/createElement.js

```diff
@@ -26,9 +26,7 @@ export default function createElement (
    * but a simple tag instead. In that case return a tagElement instance.
    */
   if (typeof element === 'string') return createTagElement(element, configs, children);
-
-  const props = omit(configs, { key: 1, ref: !element.__isForwardRef });
-
+  const props = { ...element.defaultProps, ...omit(configs, { key: 1, ref: !element.__isForwardRef })};
```

Comment:
The spread attribute might not work. Instead of babel we use buble https://github.com/bublejs/buble to bundle. (That's create smaller chunk). But I think that does not support object spread. Can you try running bundle command?

Reply:
@s-yadav the bundle command runs successfully and checked https://buble.surge.sh as well, buble works with spread operator